geode-redis/src/test/java/org/apache/geode/redis/internal/data/RedisHashTest.java

```diff
@@ -42,6 +49,15 @@ public static void beforeClass() {
     InternalDataSerializer
         .getDSFIDSerializer()
         .registerDSFID(DataSerializableFixedID.REDIS_BYTE_ARRAY_WRAPPER, ByteArrayWrapper.class);
+    // setting expiration via system properties
+    System.setProperty("redis.hscan-snapshot-cleanup-interval", "500");
+    System.setProperty("redis.hscan-snapshot-expiry", "500");
```

Comment:
In general, system properties should be managed with a `RestoreSystemProperties` rule. Also, if possible, unit tests should avoid reliance on system properties. It really turns them into integration tests. I think your previous approach of having a test-only constructor, that took these parameters, was better and I think it would be cleaner to change to that model again.

Reply:
reverted